test/cpp/interop/stress_test.cc

```diff
@@ -199,23 +214,48 @@ int main(int argc, char** argv) {
   LogParameterInfo(server_addresses, tests);
 
   WeightedRandomTestSelector test_selector(tests);
+  MetricsServiceImpl metrics_service;
 
   gpr_log(GPR_INFO, "Starting test(s)..");
 
   vector<thread> test_threads;
```

Comment:
Done. Changed std::thread to grpc::thread. Also removed "using std::vector" and "using std::pair" statements.